src/app/cube/home/home.component.html

```diff
@@ -6,7 +6,7 @@
   ></clark-blogs>
   <!-- remove after clark 5th birthday -->
   <div style="display: flex; flex-direction: row; justify-content: center;">
-    <h1 style="background: none;text-align: center; position: absolute; z-index: 10; ">HAPPY 5TH BIRTHDAY CLARK</h1>
+    <h1 style="background: none;text-align: center; position: absolute; z-index: 10; ">HAPPY 5TH BIRTHDAY <span style="color: #1C70DD">CLARK</span></h1>
```

Comment:
The banner will be removed so this hard coded color is fine.

Reply:
Maybe use the new homepage blue here? #376ED6